### DISCUSSION TITLE
Support new song length database format in SID plugin

### DISCUSSION THREAD

**Discussion by modrobert** [audacious-media-player/audacious#94] [General]
I was trying to figure out how to enable SID song length database handling (HVSC)  in Ubuntu Linux, but couldn't find any info, so checked the source code and found "SIDDATADIR/sidplayfp/", ran 'strace audacious' after that to check where the SID plugin binary in Ubuntu expected to load it, in my case it was:

```
/usr/share/sidplayfp
```

I created the directory and copied the following files there because the SID plugin was looking for them:
```
-rwxr-xr-x 1 0 0    8192 Dec  9 20:11 basic
-rwxr-xr-x 1 0 0    4096 Dec  9 20:11 chargen
-rwxr-xr-x 1 0 0    8192 Dec  9 20:11 kernal
-rw-r--r-- 1 0 0 4879153 Dec  9 20:11 Songlengths.md5
```

The C64 ROM files ('basic', 'chargen' and 'kernal') are optional I assume, but might be required for some tunes to play.

Now everything seems to work, after adding a HVSC (High Voltage SID Collection) directory to a playlist in audacious it gives the song length and handles it correctly (as opposed to looping forever for most tunes).

Perhaps this should be documented somewhere since the SID plugin settings within audacious doesn't have any config entry for it (anymore).

BTW:

I registered at the forum https://redmine.audacious-media-player.org first but no activation email is sent, tried to resend, but still failed (in case a web admin reads this).

**Comment by jlindgren90** [comment]
Thanks for documenting this :)

Gmail has been blocking activation emails from Redmine for a while now. That was one reason we set up Discussions here on GitHub. We should update the website to point here instead.
